_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -21,24 +21,56 @@ import {
 
 export const WelcomeNotice = React.createClass( {
 	displayName: 'WelcomeNotice',
+	getInitialState: function() {
+		return { showNotice: true };
+	},
+
+	dismissWelcomeNotice: function() {
+		this.setState( { showNotice: false } );
+	},
+
+	propTypes: {
+		jumpstarted: React.PropTypes.bool
+	},
+
+	getDefaultProps() {
+		return {
+			jumpstarted: false
+		};
+	},
+
+	getWelcomeMessageText: function() {
+		if ( this.props.jumpstarted ) {
+			return(
+				__( '(NEED BETTER TEXT) Great choice! By Jumpstarting your site, you have unlocked even more power of wordpress.com! {{a}}Learn more{{/a}}', {
+					components: {
+						a: <a href={ 'https://jetpack.com/support/' } target="_blank" />
+					}
+				} )
+			);
+		} else {
+			return(
+				__( 'Welcome to your Jetpack dashboard! Now you can quickly manage all of Jetpack’s great features ' +
+					'from one central location. {{a}}Learn more{{/a}}.', {
```

Comment:
I had broken this up on a first iteration when the string was way longer.  Changed it back to one line.